File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    s = s.split(' ')
    for i in range(len(s)):
        # if s[i][:2].isdigit():
        #     pass
        # else:
        s1 = s[i][:1].upper()
        s2 = s[i][1:].lower()
        s[i] = s1 + s2
    answer = ' '.join(s)
    return answer

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    bool flg = true;

    for (char c: s) {
        if (c == ' ') {
            flg = true;
        } else if (flg) {
            c = ('a' <= c && c <= 'z') ? c - ('a'-'A') : c;
            flg = false;
        } else {
            c = ('A' <= c && c <= 'Z') ? c + ('a'-'A') : c;
        }
        answer += c;
    }
    
    return answer;
}